draft-ietf-ppm-dap.md

```diff
@@ -1594,7 +1587,7 @@ encoded(struct {
     struct {
       report_metadata = struct {
         report_id = [0x0a, 0x0b, 0x0c, 0x0d, ...],
-        time = 1741986088,
+        time = 17419860,
```

Comment:
I've shaved a few significant digits off all the times and durations fairly arbitrarily. I'm open to any adjustments.